experimental/flatpak/com.gexperts.Tilix.json

```diff
@@ -0,0 +1,65 @@
+{
+        "id": "com.gexperts.Tilix",
+        "version": "master",
+        "runtime": "org.gnome.Platform",
+        "runtime-version": "3.24",
+        "sdk": "org.gnome.Sdk",
+        "command": "tilix",
+        "tags": ["Terminal", "Tile"],
+        "cleanup-commands": ["rm -rf /app/dmd",
+                             "rm -rf /app/lib"],
+        "finish-args": ["--share=ipc",
+                        "--socket=x11",
+                        "--socket=wayland",
+                        "--socket=session-bus",
+                        "--socket=pulseaudio",
+                        "--filesystem=~/.config/dconf:ro",
+                        "--filesystem=xdg-run/dconf",
+                        "--talk-name=ca.desrt.dconf",
+                        "--talk-name=org.freedesktop.Flatpak",
+                        "--own-name=com.gexperts.Tilix",
+                        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+                        "--allow=devel"],
+        "modules": [
+                {
+                        "name": "dmd",
+                        "buildsystem": "simple",
+                        "build-commands": [
+                                "cp -R dmd /app/dmd"
+                        ],
+                        "sources": [
+                                {
+                                        "type": "archive",
+                                        "url": "http://downloads.dlang.org/releases/2017/dmd.2.076.0.linux.zip",
+                                        "sha256": "7a4eea611d19323b2dda13da12aec8e7c663c484ed61a5254b388d37f7f3a57a",
+                                        "dest": "dmd"
+                                }
+                        ]
+                },
+                {
+                        "name": "tilix",
+                        "buildsystem": "simple",
+                        "build-commands": [
+                                "PATH=\"/app/dmd/linux/bin64:$PATH\" LD_LIBRARY_PATH=/app/dmd/linux/lib64 dub build --build=release",
+                                "./install.sh /app",
+                                "sed -i 's/\\/usr\\/bin\\/tilix/tilix/g' /app/share/applications/com.gexperts.Tilix.desktop",
+                                "sed -i 's/\\/usr\\/bin\\/tilix/tilix/g' /app/share/dbus-1/services/com.gexperts.Tilix.service"
+                        ],
+                        "build-options": {
+                                "build-args": ["--share=network"]
+                        },
+                        "sources": [
+                                {
+                                        "type": "git",
+                                        "url": "../../.git",
+                                        "branch": "master"
```

Comment:
This presumes that this pull is merged into master, if you want to try it change branch and/or url.